notebooks/05_operationalize/lgbm_criteo_o16n.ipynb

```diff
@@ -0,0 +1,491 @@
+{
```

Comment:
hey guys, I'm going to split this PR into two, so it gets easier to review. I will also change the name of this file to have the same naming as the one created by scott

Reply:
@jreynolds01 is not in PR https://github.com/Microsoft/Recommenders/pull/682